From 616dd298b7e9a7eb1d200ccbeaff00e8b72a1c5b Mon Sep 17 00:00:00 2001
From: Zheng Zucheng <zhengzucheng@huawei.com>
Date: Thu, 3 Jun 2021 17:09:32 +0800
Subject: [PATCH] sched: export sched_setscheduler symbol
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 616dd298b7e9a7eb1d200ccbeaff00e8b72a1c5b


hulk inclusion
category: feature
bugzilla: 55087
CVE: NA

--------------------------------

Signed-off-by: Zheng Zucheng <zhengzucheng@huawei.com>
Reviewed-by: Chen Hui <judy.chenhui@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 kernel/sched/core.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kernel/sched/core.c b/kernel/sched/core.c
index 57b236251884..7ecde2763727 100644
--- a/kernel/sched/core.c
+++ b/kernel/sched/core.c
@@ -5509,6 +5509,7 @@ int sched_setscheduler(struct task_struct *p, int policy,
 {
 	return _sched_setscheduler(p, policy, param, true);
 }
+EXPORT_SYMBOL_GPL(sched_setscheduler);
 
 int sched_setattr(struct task_struct *p, const struct sched_attr *attr)
 {
-- 
2.26.2

